test/services/CommonTaskListServiceFixture.scala

```diff
@@ -175,6 +175,29 @@ trait CommonTaskListServiceFixture {
     )
   )
 
+  val inProgressTaskSections: List[TaskListSection] = List(
+    TaskListSection(
+      sectionTitle = SectionTitle.JsaTitle,
+      taskItems = Some(List(
+        TaskListSectionItem(
+          title = TaskTitle.JSA,
+          status = TaskStatus.InProgress,
+          href = Some("http://localhost:9376/1234/jobseekers-allowance/claims")
```

Comment:
from readability point of view can we please either use a constant taxyear instead of 1234 or use proper taxyear. it would make it obvious that is a taxyear